tox.ini

```diff
@@ -15,8 +16,9 @@ envlist =
 [gh-actions]
 python =
     3.6: py36-min,py36-pypi
-    3.7: py37-min,py37-pypi,isort-check,isort-check-examples, isort-check-tests,flake8
+    3.7: py37-min,py37-pypi,flake8
     3.8: py38-min
+    3.9: py39-min,isort-check,isort-check-examples,isort-check-tests
```

Comment:
We should probably run `py38-pypi` and `py39-pypi` as well. Don't know why this was previously not done for 3.8.